using_packages/conanfile_txt.rst

```diff
@@ -98,6 +98,12 @@ requirement with one from another user or channel.
     OpenSSL/1.0.2p@conan/stable
     zlib/1.2.11@otheruser/alpha
 
+.. note::
```

Comment:
I think this note here is a bit confusing, because the user is not aware of the ``override`` keyword, it can only be set in ``conanfile.py``, not ``conanfile.txt``. Maybe this section is too introductory for this concept.

Reply:
It is true that there is not an override keyword but you are talking about overriding a requirement here too, so it might be relevant